src/configs/ScenarioParametersTabs.config.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export const SCENARIO_PARAMETERS_TABS_CONFIG = [
+  {
+    id: 0,
+    translationKey: 'commoncomponents.tab.scenario.parameters.upload.file',
+    label: 'Upload File template',
+    value: 'upload_file_template',
+    runTemplate: ['Full simulation 100 steps']
```

Comment:
Could you please change runTemplate by runTemplateIds to match with the previous comment ?